api/assets/resources.json

```diff
@@ -25,6 +25,396 @@
     "title": "SimplyRETS API",
     "description": "The SimplyRETS API is an exciting step towards making it easier for\ndevelopers and real estate agents to build something awesome with\nreal estate data!\n\nThe documentation below makes live requests to our API using the trial\ndata. To get set up with the API using live RETS data, you must have\nRETS credentials from your RETS vendor or MLS, which you can then use\nto create an app with SimplyRETS. For more information on that\nprocess, please see our [Getting\nStarted Guide](https://simplyrets.com/blog/getting-set-up.html), [FAQ](https://simplyrets.com/faq), or\n[contact us](https://simplyrets.com/\\#home-contact).\n\nBelow you'll find the API endpoints, query parameters, response bodies,\nand other information about using the SimplyRETS API. You can run\nqueries by clicking the 'Try it Out' button at the bottom of each\nsection. Check out the [help topics](/#topics) for more tips and examples on things like [IDX compliance features](/topics/fields-and-features-for-idx-compliance) and [versioning API requests](/topics/versioning-api-requests-with-media-types).\n\n### Authentication\nThe SimplyRETS API uses Basic Authentication. When you create an\napp, you'll get a set of API credentials to access your\nlistings. If you're trying out the test data, you can use\n`simplyrets:simplyrets` for connecting to the API.\n\n### Pagination\n\nTo paginate through listings, start your query with these\nparameters: 'limit=500&lastId=0'. The 'lastId' is the important\npart, you can use any limit up to 500. When you receive the\nresponse from the API with the results, check the 'Link' header for\nthe 'next' link. That link is pre-built to access the next 'page'\nof listings. Alternatively, you can use the last listing's 'mlsId'\nfrom the previous request and use that in the next query. For\nexample:\n\nFirst query:\n\ncurl -u username:password 'https://api.simplyrets.com/properties?limit=500&lastId=0'\n\nIf the 'mlsId' in the last listing of the results is '1234567', then the next query will be:\n\ncurl -u username:password 'https://api.simplyrets.com/properties?limit=500&lastId=1234567'\n\n...and so one until you have reached the final page of listings.\n\nThere a few pieces of useful information about each request stored\nin the HTTP Headers:\n\n- `X-Total-Count` shows you the total amount of listings that match\n  your current query.\n- `Link` contains pre-built pagination links for accessing the next\n'page' of listings that match your query.\n\n"
   },
+  "components": {
```

Comment:
Since analytics and properties use the same parameters, this moves the parameters from being inline in `/properties` to here, and they are `$ref`'ed from within the endpoint.

See: https://swagger.io/docs/specification/describing-parameters/ (same for v2/v3)